### ISSUE TITLE
ScrollView should unblock native scrolling when content is no longer scrollable.

### ISSUE THREAD

**Issue by trusktr** [Famous/famous#491]
For example, when you've scrolled to the bottom of a ScrollView, scrolling should now scroll the outer container which might be non-famous. This becomes tricky with nested Famous ScrollViews. Famous should detect when the outer container of a ScrollView is native, or another Famous ScrollView so it can behave accordingly (be it by event bubbling or whatever mechanism).

The bouncy effect at the end of the scrolling might need to be disabled in certain conditions for this to look nice. For example, flinging a ScrollView can show a bounce when the scroll reaches the end, but at that very moment if the user tries to continue scrolling in the same direction it should not have the elastic effect so that if the outer container is to be scrolled it will work nicely.

**Comment by MylesBorins** [comment]
closing for now as we are not going to implement low priority enhancements.  Please feel free to file a PR though
